lib/awful/rules.lua

```diff
@@ -360,6 +360,13 @@ function rules.extra_properties.placement(c, value)
     end
 end
 
+function rules.extra_properties.tags(c, value)
+    if not type(value) == "table" then return end
```

Comment:
Why a silent return instead of `error()` or `gears.debug.print_error()`? People who get this wrong would surely like some hint that something is wrong.